src/main/java/com/extendedclip/deluxemenus/menu/MenuItem.java

```diff
@@ -217,16 +213,65 @@ public ItemStack getItemStack(@NotNull final MenuHolder holder) {
             itemMeta.setDisplayName(StringUtils.color(displayName));
         }
 
-        if (!this.options.lore().isEmpty()) {
-            final List<String> lore = this.options.lore().stream()
-                    .map(holder::setPlaceholders)
-                    .map(StringUtils::color)
-                    .map(line -> line.split("\n"))
-                    .flatMap(Arrays::stream)
-                    .map(line -> line.split("\\\\n"))
-                    .flatMap(Arrays::stream)
-                    .collect(Collectors.toList());
+        // This removes all lore from the item if the option is enabled. Useful if you have a hooked item with lore that you don't want.
+        if (this.options.ignoreLore()) {
+            itemMeta.setLore(new ArrayList<>());
+        }

```

Comment:
Yes, the option is currently needed to achieve the desired result of no lore from hooked items. I just tested it and it doesn't appear to add any lore because the list will be empty. Unless we want to remove the "this.options.lore().isEmpty()" check, then we can ditch the ignoreLore option.

Reply:
You're right. I would prefer repurposing the "lore is empty" check to be used for this.

Reply:
Piggybacking off the existing lore option would be ideal, but I didn't know how that would interact so I decided to just go around it for now. I'll make the changes to allow for `lore: []` to be applied and remove the ignore_lore option.

Reply:
One thing I realized at the moment is that `lore` might be null, but when you get it from MenuItemOptions, it will be an empty list. So that will require some modification. You will have to change the entire flow from DeluxeMenusConfig to MenuItem#toItemStack.

Reply:
I take that back. It should be fine if the lore is empty instead of null. So all you need to do is remove the isEmpty check.

Reply:
I was just poking around and it doesn't seem like there is a way for backward compatibility by removing the isEmpty check. Because there is no way to check if the lore exists and if the lore is empty. So if you do `lore: []`, it's the same as just not having lore. 

This means that, as it currently works, hooked items have their lore passed on since DM doesn't touch it. Removing the isEmpty check will force the lore even if it's empty, removing the lore from the item, which breaks the current behavior. 

There would need to be another boolean that checks if lore exists vs what's in it.